src/connected-components.js

```diff
@@ -107,6 +107,12 @@ const platform = {
   setDarkMode: (value) => {
     ipcRenderer.send('pls-set-dark-setting', value)
   },
+  setFontStyle: (value) => {
```

Comment:
When you add to the connector object, please also add a PropType requirement to the schema file: https://github.com/cameronsutter/plottr_electron/blob/staging/lib/plottr_components/src/connections/pltr.js
That communicates the requirement to other projects that use the component library.